### ISSUE TITLE
rpy2 import error on Ubuntu

### ISSUE THREAD

**Issue by titipata** [daniel-acuna/reviewer_assignment#29]
Error found on Ubuntu and might be related to following anaconda issue https://github.com/ContinuumIO/anaconda-issues/issues/152

After running `celery -A reviewer_assignment_website worker`, I got following error 

``` bash
  File "/home/ubuntu/anaconda/lib/python2.7/site-packages/celery/loaders/base.py", line 298, in find_related_module
    return importlib.import_module('{0}.{1}'.format(package, related_name))
  File "/home/ubuntu/anaconda/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/ubuntu/reviewer_assignment/reviewer_assignment_website/review_scoring/tasks.py", line 4, in <module>
    import pandas.rpy.common as com
  File "/home/ubuntu/anaconda/lib/python2.7/site-packages/pandas/rpy/common.py", line 14, in <module>
    from rpy2.robjects.packages import importr
  File "/home/ubuntu/anaconda/lib/python2.7/site-packages/rpy2/robjects/__init__.py", line 15, in <module>
    import rpy2.rinterface as rinterface
  File "/home/ubuntu/anaconda/lib/python2.7/site-packages/rpy2/rinterface/__init__.py", line 101, in <module>
    from rpy2.rinterface._rinterface import *
ImportError: /home/ubuntu/anaconda/bin/../lib/libreadline.so.6: undefined symbol: PC
```

**Comment by titipata** [comment]
Solved by f48982af1969645954eb6a46e58da71360dfa526

**Comment by titipata** [comment]
Solved by f48982af1969645954eb6a46e58da71360dfa526
